docs/docs.less

```diff
@@ -119,6 +119,26 @@ dd {
   margin-left: 1em;
 }
 
+[loading] {
+  display: none;
+  text-align: center;
+
+  hx-busy {
+    font-size: 2rem;
+
+    & + * {
+      margin-top: 0.5rem;
+    }
+  }
+}
+
 [v-cloak] {
   opacity: 0;
+
+  & + [loading] {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
This CSS allows for the following...

![icons-loading](https://user-images.githubusercontent.com/545605/37495631-1648496a-287c-11e8-9672-4a9f46a1cb6c.gif)